ResultadosDigitais/matrix
# feat: save notification option on localstorage

### Description


In order to fix https://github.com/ResultadosDigitais/matrix/issues/332 we are persisting user enable/disable notification option on localstorage.

### How to test?

1. Run matrix on `augusto/save-notification-option-localstorage` branch.
- [ ] Notifications should be disabled by default.
- [ ] `notification-disabled` key shouldn't exists on localstorage.
![image](https://user-images.githubusercontent.com/9935397/125379090-48a6ea80-e366-11eb-919f-68b28bd5c923.png)

2. Enable notifications.
- [ ] `notification-disabled` key should be created on localstorage with `false` value.
![image](https://user-images.githubusercontent.com/9935397/125379520-08943780-e367-11eb-9383-ce5eae00d34f.png)
- [ ] `TOGGLE_NOTIFICATION` should be dispatched and `systemSettings.notificationDisabled` property should be updated to `false` on redux store.
- [ ] `NotificationCheckbox` component should let user disable notifications.
![image](https://user-images.githubusercontent.com/9935397/125379757-78a2bd80-e367-11eb-8c77-0e8c3fac7ff4.png)


### Expected behavior
